custom/opm/beneficiary.py

```diff
@@ -489,6 +489,24 @@ def child_received_ors(self):
             return True
 
     @property
+    def child_received_ors_in_this_window(self):
+        months = self.child_age % 3
+        if months == 0:
+            months_before = 3
+        elif months == 1:
+            months_before = 1
+        else:
+            months_before = 2
```

Comment:
can you explain these numbers, maybe with a code comment? i don't follow the logic